velox/docs/configs.rst

```diff
@@ -206,6 +211,12 @@ Spilling
        If the limit is zero, then the spiller always spills a previously spilled partition if it has any data. This is
        to avoid spill from a partition with a small amount of data which might result in generating too many small
        spilled files.
+   * - spill_compression_codec
+     - string
+     - none
+     - Specifies the compression algorithm type to compress the spilled data before write to disk to trade cpu for
+       spilling io efficiency. The supported compression kind includes 'zlib', 'snappy', 'lzo', 'zstd', 'lz4' and
+       'gzip'. If it set to 'none', then there is no compression.g
```

Comment:
Extra 'g' at the end.

"If it set to 'none', then there is no compression" -> "NONE means no compression."

Reply:
Thanks!